### ISSUE TITLE
[Documentation] Quick Start step 7 does not work: "evennia" not a valid command

### ISSUE THREAD

**Issue by spiralofhope** [evennia/evennia#2016]
#### Documentation issue

https://github.com/evennia/evennia/wiki/Getting-Started#quick-start

``` bash
evennia --init mygame
```

This command fails, because there is no command called `evennia`.

When I try manually running:

``` bash
evennia/bin/unix/evennia
```

I get:

```
Traceback (most recent call last):
  File "evennia/bin/unix/evennia", line 9, in <module>
    from  evennia.server.evennia_launcher import main
ImportError: No module named evennia.server.evennia_launcher
```

I am following the steps exactly.

**Comment by Griatch** [comment]
This happens if the install step of Evennia failed not performed or failed for some reason. In the activated virtualenv,  the Evennia package must be installed with `pip install -e evennia` before the `evennia` commands becomes available on the command line. The error from running the binary directly is due to the same reason. 
Check if you get any errors when installing. 

If the issue persists,  please make sure your virtualenv is active and inspect/post the installed packages with `pip list`.

**Comment by spiralofhope** [comment]
The previous command did work (with `pip3`) with no error:

``` bash
pip3  install  -e  evennia
```

`(evenv)` was showing.

I'll learn how to delete things and start over to give more information.

**Comment by spiralofhope** [comment]
Things seem a little different this time, because using `evennia --init mygame` didn't work.

But here is a complete log of my fresh attempt:

```
prompt:~/l/mud$ git  clone  https://github.com/evennia/evennia.git
Cloning into 'evennia'...
remote: Enumerating objects: 67, done.
remote: Counting objects: 100% (67/67), done.
remote: Compressing objects: 100% (56/56), done.
remote: Total 45763 (delta 19), reused 33 (delta 11), pack-reused 45696
Receiving objects: 100% (45763/45763), 36.87 MiB | 3.16 MiB/s, done.
Resolving deltas: 100% (34282/34282), done.
prompt:~/l/mud$ virtualenv  evenv
Running virtualenv with interpreter /usr/bin/python2
New python executable in /home/user/l/mud/evenv/bin/python2
Also creating executable in /home/user/l/mud/evenv/bin/python
Installing setuptools, pkg_resources, pip, wheel...done.
prompt:~/l/mud$ source  evenv/bin/activate
(evenv) prompt:~/l/mud$ pip3  install  -e  mygame
mygame should either be a path to a local project or a VCS url beginning with svn+, git+, hg+, or bzr+
(evenv) prompt:~/l/mud$ pip3  install  -e  evennia
Obtaining file:///home/user/l/mud/evennia
Requirement already satisfied: anything in /home/user/.local/lib/python3.7/site-packages (from evennia==0.9.0) (0.2.1)
Requirement already satisfied: autobahn>=17.9.3 in /home/user/.local/lib/python3.7/site-packages (from evennia==0.9.0) (19.11.2)
Requirement already satisfied: black in /home/user/.local/lib/python3.7/site-packages (from evennia==0.9.0) (19.10b0)
Requirement already satisfied: django-sekizai in /home/user/.local/lib/python3.7/site-packages (from evennia==0.9.0) (1.0.0)
Requirement already satisfied: django<2.3,>=2.2.5 in /home/user/.local/lib/python3.7/site-packages (from evennia==0.9.0) (2.2.9)
Requirement already satisfied: inflect in /home/user/.local/lib/python3.7/site-packages (from evennia==0.9.0) (4.0.0)
Requirement already satisfied: mock>=1.0.1 in /home/user/.local/lib/python3.7/site-packages (from evennia==0.9.0) (3.0.5)
Requirement already satisfied: model_mommy in /home/user/.local/lib/python3.7/site-packages (from evennia==0.9.0) (2.0.0)
Requirement already satisfied: pytz in /home/user/.local/lib/python3.7/site-packages (from evennia==0.9.0) (2019.3)
Requirement already satisfied: twisted<20.0.0,>=19.2.1 in /home/user/.local/lib/python3.7/site-packages (from evennia==0.9.0) (19.10.0)
Requirement already satisfied: cryptography>=2.7 in /home/user/.local/lib/python3.7/site-packages (from autobahn>=17.9.3->evennia==0.9.0) (2.8)
Requirement already satisfied: six>=1.11.0 in /usr/lib/python3/dist-packages (from autobahn>=17.9.3->evennia==0.9.0) (1.12.0)
Requirement already satisfied: txaio>=18.8.1 in /home/user/.local/lib/python3.7/site-packages (from autobahn>=17.9.3->evennia==0.9.0) (18.8.1)
Requirement already satisfied: appdirs in /home/user/.local/lib/python3.7/site-packages (from black->evennia==0.9.0) (1.4.3)
Requirement already satisfied: pathspec<1,>=0.6 in /home/user/.local/lib/python3.7/site-packages (from black->evennia==0.9.0) (0.7.0)
Requirement already satisfied: regex in /home/user/.local/lib/python3.7/site-packages (from black->evennia==0.9.0) (2020.1.8)
Requirement already satisfied: toml>=0.9.4 in /home/user/.local/lib/python3.7/site-packages (from black->evennia==0.9.0) (0.10.0)
Requirement already satisfied: typed-ast>=1.4.0 in /home/user/.local/lib/python3.7/site-packages (from black->evennia==0.9.0) (1.4.0)
Requirement already satisfied: click>=6.5 in /home/user/.local/lib/python3.7/site-packages (from black->evennia==0.9.0) (7.0)
Requirement already satisfied: attrs>=18.1.0 in /home/user/.local/lib/python3.7/site-packages (from black->evennia==0.9.0) (19.3.0)
Requirement already satisfied: django-classy-tags>=0.3.1 in /home/user/.local/lib/python3.7/site-packages (from django-sekizai->evennia==0.9.0) (0.9.0)
Requirement already satisfied: sqlparse in /home/user/.local/lib/python3.7/site-packages (from django<2.3,>=2.2.5->evennia==0.9.0) (0.3.0)
Requirement already satisfied: importlib-metadata; python_version < "3.8" in /home/user/.local/lib/python3.7/site-packages (from inflect->evennia==0.9.0) (1.4.0)
Requirement already satisfied: zope.interface>=4.4.2 in /home/user/.local/lib/python3.7/site-packages (from twisted<20.0.0,>=19.2.1->evennia==0.9.0) (4.7.1)
Requirement already satisfied: incremental>=16.10.1 in /home/user/.local/lib/python3.7/site-packages (from twisted<20.0.0,>=19.2.1->evennia==0.9.0) (17.5.0)
Requirement already satisfied: hyperlink>=17.1.1 in /home/user/.local/lib/python3.7/site-packages (from twisted<20.0.0,>=19.2.1->evennia==0.9.0) (19.0.0)
Requirement already satisfied: PyHamcrest>=1.9.0 in /home/user/.local/lib/python3.7/site-packages (from twisted<20.0.0,>=19.2.1->evennia==0.9.0) (1.9.0)
Requirement already satisfied: Automat>=0.3.0 in /home/user/.local/lib/python3.7/site-packages (from twisted<20.0.0,>=19.2.1->evennia==0.9.0) (0.8.0)
Requirement already satisfied: constantly>=15.1 in /home/user/.local/lib/python3.7/site-packages (from twisted<20.0.0,>=19.2.1->evennia==0.9.0) (15.1.0)
Requirement already satisfied: cffi!=1.11.3,>=1.8 in /home/user/.local/lib/python3.7/site-packages (from cryptography>=2.7->autobahn>=17.9.3->evennia==0.9.0) (1.13.2)
Requirement already satisfied: zipp>=0.5 in /home/user/.local/lib/python3.7/site-packages (from importlib-metadata; python_version < "3.8"->inflect->evennia==0.9.0) (0.6.0)
Requirement already satisfied: setuptools in /usr/lib/python3/dist-packages (from zope.interface>=4.4.2->twisted<20.0.0,>=19.2.1->evennia==0.9.0) (40.8.0)
Requirement already satisfied: idna>=2.5 in /usr/lib/python3/dist-packages (from hyperlink>=17.1.1->twisted<20.0.0,>=19.2.1->evennia==0.9.0) (2.6)
Requirement already satisfied: pycparser in /home/user/.local/lib/python3.7/site-packages (from cffi!=1.11.3,>=1.8->cryptography>=2.7->autobahn>=17.9.3->evennia==0.9.0) (2.19)
Requirement already satisfied: more-itertools in /home/user/.local/lib/python3.7/site-packages (from zipp>=0.5->importlib-metadata; python_version < "3.8"->inflect->evennia==0.9.0) (8.1.0)
Installing collected packages: evennia
  Found existing installation: evennia 0.9.0
    Uninstalling evennia-0.9.0:
      Successfully uninstalled evennia-0.9.0
  Running setup.py develop for evennia
Successfully installed evennia
(evenv) prompt:~/l/mud$ evennia  --init  evennia
bash: evennia: command not found
```

(The instructions for the full Linux install worked for me, but the quick start guide didn't when I tried it.)

**Comment by Griatch** [comment]
I think the issue here is that pip3 is _not_ in your virtualenv so it installs to `/home/user/.local/lib/python3.7/...` instead (as shown by the paths above) rather than into your virtualenv folder. 

As mentioned in another of these tickets, you use just plain `pip` inside your virtualenv (not pip3), you just need to make sure the virtualenv uses python3.7 (set it up with `virtualenv -p /path/to/python3.7 evenv` if your system still defaults to py2). 

So, to summarize, you seem to have installed to your `/home/.local`. Most likely the `evennia` command will work _outside_ your virtualenv now.

**Comment by spiralofhope** [comment]
I'm going to close this because I don't understand the topic well enough to fix the documentation for a Debian 10 user to install using the quick start guide. (since I got it working using a variation of the full install instructions)
